css/login.css

```diff
@@ -0,0 +1,120 @@
+body{
+	margin: 0;
+
+	background-color: #555555;
+	font-family: 'Roboto', sans-serif;
+}
+
+.section{
+	height: 640px;
+}
+
+.section1{
+	background-image: url('../img/feed.jpg');
+}
+
+.logInBox{
+	height: 400px;
+	text-align:center;
+	align-self: center;
+
+	box-shadow: 0 0 8px black;
+	border-radius: 3px;
+	background-color: #F5F5F5;
+}
+
+.logInBoxHead{
+	line-height: 75px;
+
+	background-color: #1ab2ff;
+	box-shadow: 0 3px 10px grey;
+	color: white;
+	font-size: 26px;
+	
+}
+
+.logInBoxContent{
+	width: 90%;
+	margin: 0 auto;
+	margin-top: 20px;
+}
+
+.logInForm{
+	margin-top: 20px;
+	text-align: center;
+
+	background-color: #F5F5F5;
+}
+
+.logInFormInput{
+	width: 100%;
+	height: 50px;
+	padding-left: 10px;
+	margin-bottom: 10px;
+
+	border: 0;
+	border-bottom: 1px solid grey;
+	background-color: inherit;
+	font-size: 15px;
+}
+
+.logInFormButton{
+	width: 120px;
+	height: 30px;
+	margin: 0 auto;
+	margin-top: 50px;
```

Comment:
`50px auto 0`, rather, because you need to account for the bottom margin as well, which in your case would be 0. `50px auto` would mean `50px` for the top and bottom and `auto` for left and right.